lib/salus/scanners/lang_version.rb

```diff
@@ -0,0 +1,75 @@
+require 'salus/scanners/base'
+require 'json'
+
+module Salus::Scanners
+  class LangVersion < Base
+    class SemVersion < Gem::Version; end
+
+    def run
+      version = SemVersion.new(lang_version)
+      min_version = SemVersion.new(@config['min_version']) if @config['min_version']
+      max_version = SemVersion.new(@config['max_version']) if @config['max_version']
+
+      if min_version && (version < min_version)
+        msg = "Repository language version (#{version}) is less " \
+        "than minimum configured version (#{min_version})"
+        log(msg)
+        return report_failure
+      end
+
+      if max_version && (version > max_version)
+        msg = "Repository language version (#{version}) is greater " \
+        "than maximum configured version (#{max_version})"
+        log(msg)
+        return report_failure
+      end
+
+      report_success
+    end
+
+    def self.supported_languages
+      %w[go ruby]
+    end
+
+    def should_run?
+      go_project? || ruby_project?
+    end
+
+    private
+
+    def lang_version
+      return go_version if go_project?
+      return ruby_version if ruby_project?
+
+      nil
+    end
+
+    def go_version
+      shell_return = Dir.chdir(@repository.path_to_repo) do
+        cmd = "go mod edit -json"
+        run_shell(cmd)
+      end
+
+      shell_return_json = JSON.parse(shell_return.stdout)
+
+      shell_return_json["Go"]
+    end
+
+    def go_project?
+      @repository.go_mod_present?
+    end
+
+    def ruby_version
```

Comment:
Small ask - can you document what the cmd is returning?  Sample output etc.

Reply:
Documented as comment: https://github.com/coinbase/salus/pull/493/files#diff-0f60519da2f31b11e367c601ab88f4482a411d5e43130b3c840af99bc114da0fR19